### PR TITLE
feature: blog post Meet Nukomeeter: Adam

### PR DESCRIPTION
**Pull request by myviridis**

### CODE REVIEW

File: content/blog/2015-07_Meet Nukomeeter  Adam.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: interview
```
Comment (tomkuk) [inline content/blog/2015-07_Meet Nukomeeter  Adam.md]: @myviridis  I've got compilation error here , this is wrong format for tags. It looks like authors above

Comment (myviridis) [inline content/blog/2015-07_Meet Nukomeeter  Adam.md]: I think it tag "interview" written in a wrong way. I've corrected it,
we'll see.

*Kasia Popek*

Communications Manager at Nukomeet


___


You can contact me at

mail *kasia@nukomeet.com*

mobile *+48 609 160 095*


Visit our website *www.nukomeet.com*




On Thu, Jul 9, 2015, at 09:17 PM, Tomek Kuklis wrote:
> In content/blog/2015-07_Meet Nukomeeter  Adam.md[1]:


> > @@ -0,0 +1,40 @@
> +--- +created_at: 2015-07-02 +kind: article +publish: true +title:
> "Meet Nukomeeter: Adam" +authors: +- adam +- kasia +tags: interview
> @myviridis[2]  I've got compilation error here , this is wrong format
> for tags. It looks like authors above


> — Reply to this email directly or view it on GitHub[3].


>



Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/192#discussion_r34292475
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/192/files#r34292475

**Comment by myviridis** [comment]
Should be ok now. Can you confirm?

File: content/blog/2015-07_Meet Nukomeeter  Adam.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+created_at: 2015-07-02
+kind: article
+publish: true
+title: "Meet Nukomeeter: Adam"
+authors:
+- adam
+- kasia
+tags: 
+- interview
```
Comment (2can) [inline content/blog/2015-07_Meet Nukomeeter  Adam.md]: @myviridis possible tags are ['business', 'programming', 'innovation', 'society', 'technology', 'web', 'meetup'] 

they are defined in https://github.com/nukomeet/nukomeet.com/blob/master/Rules

Comment (myviridis) [inline content/blog/2015-07_Meet Nukomeeter  Adam.md]: Improved

*Kasia Popek*

Communications Manager at Nukomeet


___


You can contact me at

mail *kasia@nukomeet.com*

mobile *+48 609 160 095*


Visit our website *www.nukomeet.com*




On Thu, Jul 9, 2015, at 10:13 PM, Bartłomiej Wiaduch wrote:
> In content/blog/2015-07_Meet Nukomeeter  Adam.md[1]:


> > @@ -0,0 +1,41 @@
> +--- +created_at: 2015-07-02 +kind: article +publish: true +title:
> "Meet Nukomeeter: Adam" +authors: +- adam +- kasia +tags: +- interview
> @myviridis[2] possible tags are ['business', 'programming',
> 'innovation', 'society', 'technology', 'web', 'meetup']


> they are defined in
> https://github.com/nukomeet/nukomeet.com/blob/master/Rules


> — Reply to this email directly or view it on GitHub[3].


>


Links:

  1. https://github.com/nukomeet/nukomeet.com/pull/192#discussion_r34298163
  2. https://github.com/myviridis
  3. https://github.com/nukomeet/nukomeet.com/pull/192/files#r34298163

**Pull request merged**